lua/nvim-tree/renderer.lua

```diff
@@ -302,9 +301,13 @@ local function update_draw_data(tree, depth, markers)
         table.insert(hl, {'NvimTreeImageFile', index, offset+#icon+#git_icons, -1 })
       end
 
-      if vim.g.nvim_tree_highlight_opened_files then
-        if vim.fn.bufloaded(node.absolute_path) > 0 then
+      if vim.fn.bufloaded(node.absolute_path) > 0 then
```

Comment:
hum i wouldn't call bufloaded for each iteration, could you maybe do a should_hl_opened_files variable based on the value of nvim_tree_highlight_opened_files ? Done once at module initialisation (outside of the function), it would be faster i think. Maybe just make a function generator like i did at the start to handle this code.

Reply:
Hmm, I see. That's neat! I'll work on that